src/app/modules/app/pages/classrooms/components/classroom-form/classroom-form.component.ts

```diff
@@ -199,6 +205,12 @@ export class ClassroomFormComponent implements OnInit {
     } else {
       this.classroomService.create(classroom).subscribe(this.requestHandler);
     }
+
```

Comment:
In the lines above you make the create or edit requests, and down here you are making the create request another time and even when the form is meant to edit the classroom, fix that please